src/server/components/delete-modal.jsx

```diff
@@ -15,7 +14,8 @@ export default class DeleteModal extends React.Component {
   deleteObject() {
     // add delete here
 
-    deleteFileOnServer(this.props.elementID)
+    this.props
+      .deleteFunction(this.props.elementID)
```

Comment:
by moving to `this.props.deleteFunction` we can test the actual deleting function independently, and we don't have to pass around boilerplate url strings